### ISSUE TITLE
Error when dashboard is deleted manually.

### ISSUE THREAD

**Issue by gkocur** [suzuki-shunsuke/go-graylog#230]
I decided to open new issue since I think it's not related to the support of all widget types.

## Environment

* Terraform version: 0.12.8
* Graylog version: 3.1
* terraform-provider-graylog or go-graylog version: 10.0.0-0
 
## Overview

When the dashboard and widgets are created using tf and then the dashboard is deleted manually the `terraform plan` exits with error. I remember a similar issue was already fixed for streams/stream rules.

## Expected behavior

The dashboard and widgets should be recreated

## Actual behavior

```
Error: graylog API error: GET https://graylog.monitoring.flexys.co.uk/api/dashboards/5e20791e2839880026cb5399/widgets/869667a7-6ff2-48cd-9266-f0befe745a99 404: {"type":"ApiError","message":"Couldn't find dashboard with ID 5e20791e2839880026cb5399"}
```

## How to reproduce

-  create a dashboard with terraform
-  delete if manually
- run `terraform plan`

**Comment by suzuki-shunsuke** [comment]
Related issue: https://github.com/suzuki-shunsuke/go-graylog/issues/154

Thank you for your feedback.
I'll fix.

**Comment by suzuki-shunsuke** [comment]
Oh, unfortunately Get Dashboard Widget API returns not 404 but 500. `GET /dashboards/{dashboardId}/widgets/{widgetId}`.
We can't distinguish whether the dashboard widget was removed or the system trouble occurs...
http://127.0.0.1:9000/api/api-browser/#!/Dashboards/Widgets/getWidget_get_3

![image](https://user-images.githubusercontent.com/13323303/72604735-d6b33800-395e-11ea-8a94-ec1bb3d61691.png)

**Comment by suzuki-shunsuke** [comment]
I decided to accept status codes except for 404.
https://github.com/suzuki-shunsuke/go-graylog/pull/231/files

**Comment by suzuki-shunsuke** [comment]
https://github.com/suzuki-shunsuke/go-graylog/pull/231 is merged.

**Comment by suzuki-shunsuke** [comment]
I have released [v10.0.1](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v10.0.1).

**Comment by gkocur** [comment]
Will test it asap, thank you!

**Comment by gkocur** [comment]
I still get error with 10.0.1:
```
Error: graylog API error: GET https://graylog.domain.com/api/dashboards/5e21964d6c62720027c9ba90 404: {"type":"ApiError","message":"Couldn't find dashboard with ID 5e21964d6c62720027c9ba90"}
```

**Comment by suzuki-shunsuke** [comment]
Oh, sorry. I got it.
This is a bug of `graylog_dashboard_widget_positions`.
I'll fix.

**Comment by suzuki-shunsuke** [comment]
https://github.com/suzuki-shunsuke/go-graylog/pull/233

**Comment by suzuki-shunsuke** [comment]
I have released [v10.0.2-0](https://github.com/suzuki-shunsuke/go-graylog/releases/tag/v10.0.2-0).
Please check.

I should have released not v10.0.1 but v10.0.1-0 ...

**Comment by gkocur** [comment]
10.0.2-1 works as expected, you are amazing!
